velox/exec/Trace.h

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
```

Comment:
Thank you @czentgr for the review.   Added a commit for the re-order.
I am not sure if #11352 is active and when it will be merged.
I am testing the Gluten on Ubuntu 24.04. If the code change can be merged quickly,  I am able to start the E2E test as early as possible.  Thanks.